Podfile

```diff
@@ -11,13 +11,13 @@ workspace 'WordPress.xcworkspace'
 ##
 def wordpress_shared
     ## for production:
-    pod 'WordPressShared', '~> 1.8.9'
+    #pod 'WordPressShared', '~> 1.8.9'
 
     ## for development:
     # pod 'WordPressShared', :path => '../WordPress-iOS-Shared'
 
     ## while PR is in review:
-    # pod 'WordPressShared', :git => 'https://github.com/wordpress-mobile/WordPress-iOS-Shared', :branch => ''
+    pod 'WordPressShared', :git => 'https://github.com/wordpress-mobile/WordPress-iOS-Shared', :branch => 'issue/add-deleted-orphan-event'
```

Comment:
I just realized we probably should not have merged this until https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/227 was ready. I'm trying to update WordPressShared to the next beta version for some other new changes that are required, and now we're stuck pointing to a branch until that PR is resolved.

Reply:
Hi @koke, sorry about that.

Here we go: https://github.com/wordpress-mobile/WordPress-iOS/pull/13081

Reply:
Gah! 🤦‍♂ That's my bad. I totally forgot about the pod. Sorry, everyone!